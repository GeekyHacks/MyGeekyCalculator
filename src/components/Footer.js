import Calculator from './Calculator';

const Footer = () => {
  const result = (
    <div>
      <footer>
        <h3 className="copyRights">
          Copy rights reserved to
          <a href={Calculator}> GeekyHacks</a>
        </h3>
      </footer>
    </div>
  );

  return result;
};

export default Footer;
