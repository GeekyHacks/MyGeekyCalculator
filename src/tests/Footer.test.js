import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../components/Footer';

test('renders Footer correctly', () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
