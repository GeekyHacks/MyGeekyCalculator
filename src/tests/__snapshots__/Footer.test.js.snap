// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Focorrectly 1`] = `
<div>
  <div>
    <footer>
      <h3
        class="copyRights"
      >
        Copy rights reserved to
        <a>
           GeekyHacks
        </a>
      </h3>
    </footer>
  </div>
</div>
`;

exports[`renders Footer correctly 1`] = `
<div>
  <div>
    <footer>
      <h3
        class="copyRights"
      >
        Copy rights reserved to
        <a>
           GeekyHacks
        </a>
      </h3>
    </footer>
  </div>
</div>
`;

exports[`renders correctly 1`] = `
<div>
  <div
    id="CalculatorContainer"
  >
    <div
      id="CalcInput"
    >
      0
    </div>
    <div
      id="Btns"
    >
      <button
        type="button"
      >
        AC
      </button>
      <button
        type="button"
      >
        +/-
      </button>
      <button
        type="button"
      >
        %
      </button>
      <button
        class="operator"
        type="button"
        value="+"
      >
        รท
      </button>
      <button
        type="button"
      >
        7
      </button>
      <button
        type="button"
      >
        8
      </button>
      <button
        type="button"
      >
        9
      </button>
      <button
        class="operator"
        type="button"
        value="+"
      >
        x
      </button>
      <button
        type="button"
      >
        4
      </button>
      <button
        type="button"
      >
        5
      </button>
      <button
        type="button"
      >
        6
      </button>
      <button
        class="operator"
        type="button"
        value="+"
      >
        -
      </button>
      <button
        type="button"
      >
        1
      </button>
      <button
        type="button"
      >
        2
      </button>
      <button
        type="button"
      >
        3
      </button>
      <button
        class="operator"
        type="button"
        value="+"
      >
        +
      </button>
      <button
        id="zero"
        type="button"
        value="0"
      >
        0
      </button>
      <button
        type="button"
      >
        .
      </button>
      <button
        class="operator"
        type="button"
        value="="
      >
        =
      </button>
    </div>
  </div>
</div>
`;
